README.md

```diff
@@ -19,11 +19,11 @@ When writing Python code for service mapping or other NSO development Pylint is
 Since usually you build packages as part of a CI job with that the Pylint will also check the Python code. With that you prevent that syntactic errors or code ends up in a merge request since for a successful pipeline run you have to fix them.
```

Comment:
This sentence doesn't sound too well, can you refactor the "_...a CI job with that the Pylint..._"